### PR TITLE
Describe anonymous access, system:authenticated group

### PR DESCRIPTION
**Pull request by liggitt**
Docs for https://github.com/kubernetes/kubernetes/pull/32386

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1342)

### CODE REVIEW

**Comment by liggitt** [comment]
kubernetes/kubernetes#32386 has merged, @ericchiang PTAL

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -363,6 +366,22 @@ Please refer to the [discussion](https://github.com/kubernetes/kubernetes/pull/1
...
+When enabled, requests that are not rejected by other configured authentication methods are 
+treated as anonymous requests, and given a username of `system:anonymous` and a group of 
+`system:unauthenticated`.
+
+For example, on a server with token authentication configured, and anonymous access enabled,
+a request providing an invalid bearer token would receive a `401 Unauthorized` error. 
+A request providing no bearer token would be treated as an anonymous request. 
+
+If you rely on authentication alone to authorize access, either change to use an 
+authorization mode other than `AlwaysAllow`, or or set `--anonymous-auth=false`.
```
Comment (ericchiang) [inline docs/admin/authentication.md]: s/or or/or/g

**Comment by liggitt** [comment]
Nothing changed with the unauthenticated port. It still does not hook up the authn or authz chains.

**Comment by ericchiang** [comment]
> Nothing changed with the unauthenticated port. It still does not hook up the authn or authz chains.
 Show all checks

Oh, nevermind then. lgtm pending the typo.

**Pull request merged**